BUILDTARGETS.md

```diff
@@ -2,8 +2,35 @@
 
 The Unity Buy SDK currently supports the following build targets:
 
+* [Android](#Android)
 * [iOS](#ios)
 
+# Android
+
+### Requirements
+
+The Unity Buy SDK requires Android games to have a minimum API level of Android 4.4 'Kit Kat' (API Level 19)
+
+### Building
+
+The library comes in a form of a `.AAR` file. In doing so we have provided a [mainTemplate.gradle](Assets/Plugins/Android/mainTemplate.gradle) that will be used by Unity when creating a Gradle project.
```

Comment:
Let's try to remember our target demographic here. These are Unity devs and not mobile devs. Up until this point I've read the requirements and that's all cool. But immediately I get told about `.AAR` files and `.gradle` files things I might not know anything about.

Personally I thin it'd be handy to have a step by step implementation guide where I don't need to learn about this stuff unless I want to learn more. Same sort of goes for iOS 😬